docker-compose-mysql8.yml

```diff
@@ -0,0 +1,19 @@
+version: '3'
+services:
+  zookeeper:
+    build: ./zookeeper
```

Comment:
each *-compose file builds zookeeper. Some tests require it. We have custom zk because of this: https://github.com/eventuate-foundation/eventuate-common/blob/master/zookeeper/zoo.cfg#L29

Reply:
Great but is there an alternative to copy pasting the ZK container repeatedly?

Reply:
I can create a separate compose file for zookeeper

Reply:
I believe the 'extendedTypesEnabled' config property in zoo.cfg has no effect because it could only be enabled by Java system property only.

example
  zookeeper:
    image: ./zookeeper
    ports:
      - 2181:2181
    environment:
      - JVMFLAGS=-Dzookeeper.extendedTypesEnabled=true -Dzookeeper.emulate353TTLNodes=true

zookeeper.extendedTypesEnabled
(Java system property only: zookeeper.extendedTypesEnabled)

New in 3.5.4, 3.6.0: Define to "true" to enable extended features such as the creation of TTL Nodes. They are disabled by default. IMPORTANT: when enabled server IDs must be less than 255 due to internal limitations.